templates/default/client.rb.erb

```diff
@@ -40,7 +40,7 @@ https_proxy "<%= node["chef_client"]["config"]["https_proxy"] %>"
 no_proxy "<%= node["chef_client"]["config"]["no_proxy"] %>"
 <% end -%>
 
-<% if node.attribute?("ohai") && node["ohai"].attribute?("plugin_path") -%>
+<% unless node["ohai"]["plugin_path"].nil? -%>
```

Comment:
Yep that would totally work. I was mostly just keeping things consistent with the other checks in the same template.